ansible/configs/sap-integration/files/k8s/ns_fuse.j2

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
Why is this a jinja file? There is nothing being replaced in here. You can just use `lookup('file'...` when you actually copy it.

Reply:
Yes, I know, but for consistency and simplicity, I'm using the same approach for every file even there are no variables to be replaced. Some of these files have variables, so adding variables in the future for the ones that don't have any right now will be quite easy and won't need to change the task. This way every new object created out from files on that k8s directory are added the same way.